### PR TITLE
[Docs/Building.md] Added some Arch Linux specific build instructions, no change to cppcheck as I haven't tested it yet.

### PR DESCRIPTION
**Pull request by DirtyCasting**
Spent a few minutes trying to compile, ran into an issue with CMake not finding my openssl install.  This just adds the command to install dependencies, ninja, and a one-line change under configuration for a quick fix to the issue.

### CODE REVIEW

File: Docs/Building.md [hunk 1]
```diff
@@ -108,6 +109,7 @@ For the `OPENSSL_ROOT_DIR` parameter, set the directory for where ever the opens
 
 - macOS: `/usr/local/opt/openssl` or otherwise depending on your setup (if you're using HomeBrew, MacPorts or installed in from source)
 - Linux: This parameter is not necessary on linux. (CMake can find it on it's own)
+- > **Note for Arch Linux Users:** CMake has trouble finding `libssl` and `libcrypto` by defeault.  Specifying them with `-DOPENSSL_SSL_LIBRARY=/usr/lib/libssl.so.1.1` and `-DOPENSSL_CRYPTO_LIBRARY=/usr/lib/libcrypto.so.1.1` seems to fix the issue.
```
Comment (poco0317) [inline Docs/Building.md:112]: `defeault` -> `default`

Comment (poco0317) [reply Docs/Building.md:112]: Also consider rewording and/or instead using full code blocks to try to get these commands to not be split across lines.

**Review by poco0317 — CHANGES_REQUESTED** [review]
The base of the PR should be the develop branch. It is worth testing to see if building works without these extra instructions on a very recent checkout of develop as well, since we changed the openssl cmake just a little.

**Comment by jameskr97** [comment]
Huh, Déjà vu. Try develop. Arch should work on develop without anything extra *I think*.

**Comment by Ulti-FD** [comment]
Haven't had any issues as of recent on develop

**Comment by jameskr97** [comment]
> Haven't had any issues as of recent on develop

Yeah, I've been away from the project for a little while, but I definitely remember coming across this and fixing it.

**Comment by poco0317** [comment]
superceded by current develop changes and the existence of the AUR package `etterna`

reopen if issue is present on latest develop branch and the requested changes are made

**Pull request closed without merging**